uReplicator-Common/src/main/java/com/uber/stream/kafka/mirrormaker/common/utils/HttpClientUtils.java

```diff
@@ -79,4 +103,28 @@ public static int postData(final HttpClient httpClient,
     return httpClient.execute(httpPost, HttpClientUtils.createResponseCodeExtractor());
   }
 
+  public static int deleteData(final HttpClient httpClient,
+      final RequestConfig requestConfig,
+      final String host,
+      final int port,
+      final String topicName,
+      final String src,
+      final String dst,
+      final int routeId) throws IOException, URISyntaxException {
+    URI uri = new URIBuilder()
+        .setScheme("http")
+        .setHost(host)
+        .setPort(port)
+        .setPath("/topics/" + topicName)
```

Comment:
Since it is common utils, make it more generic using "path" and construct it by caller instead.